iothub/device/src/Common/Exceptions/IotHubThrottledException.cs

```diff
@@ -8,8 +8,12 @@
 namespace Microsoft.Azure.Devices.Client.Exceptions
 {
     /// <summary>
-    /// The exception that is thrown when the service requires exponential back-off because it has exceeded the maximum number of allowed active requests.
+    /// This exception is thrown when the requests to the IoT hub exceed the limits based on the tier of the hub.
+    /// Retrying with exponential back-off could resolve this error.
```

Comment:
nit: do you think it would be useful to inform users why retrying with exponential backoff with jitter is more useful than retrying with constant time?

Reply:
It could be but it might also be information overload. But this is all very dependent on the use-case of the customer and what IoT hub service does (which we ourselves don't know). I'm inclined to leave it this way and see if customers really ask for more information.